conf/modules.config

```diff
@@ -220,6 +243,28 @@ process {
         ext.prefix = { "${meta.id}_run${meta.run}_lambdafiltered" }
     }
 
+    withName: CHOPPER {
+        ext.args2  = [
+            !params.keep_lambda ? "--contam ${params.lambda_reference}": "",
+            params.longreads_min_quality ? "--quality ${params.longreads_min_quality}": "",
+            params.longreads_min_length ? "--minlength ${params.longreads_min_length}": "",
+        ].join(' ').trim()
```

Comment:
I added the option to use chopper for filtering. Since the conditions in the subworkflow got a bit messy, i decided to put the logic in the modules.config. In this way if params.keep_phage is true, and params.longread_filtering_tool is chopper will run both phage removal and filtering. what do you think of this @jfy133?